docs/docsite/rst/porting_guides/porting_guide_12.rst

```diff
@@ -1235,10 +1329,17 @@ google.cloud
 grafana.grafana
 ^^^^^^^^^^^^^^^
 
-- Ability to set custom directory path for *.alloy config files by @voidquark in https://github.com/grafana/grafana-ansible-collection/pull/294
+- Ability to set custom directory path for \*.alloy config files by @voidquark in https://github.com/grafana/grafana-ansible-collection/pull/294
```

Comment:
This backslash looks weird, but I might be wrong there.

Reply:
It's OK, same as in https://github.com/ansible/ansible-documentation/pull/3108#discussion_r2411236207.